client/js/autocompletion.js

```diff
@@ -214,29 +216,21 @@ function enableAutocomplete(input) {
 		"keydown"
 	);
 
-	const editor = new Textarea(input);
-	const textcomplete = new Textcomplete(editor, {
-		dropdown: {
-			className: "textcomplete-menu",
-			placement: "top",
-		},
-	});
-
-	textcomplete.register([
+	const strategies = [
 		emojiStrategy,
 		nicksStrategy,
 		chanStrategy,
 		commandStrategy,
 		foregroundColorStrategy,
 		backgroundColorStrategy,
-	]);
+	];
 
-	// Activate the first item by default
```

Comment:
Having tested on 4.3.0, I don't think this does anything anymore, and the issue is hidden / not archived on Wayback Machine, so I removed this code. The Textcomplete types don't point to `dropdown` valid anymore.